app/lib/currencies_loader.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module CurrenciesLoader
+  DEFAULT_CURRENCIES = {
+    'USD - American Dollar' => 'USD'
+  }.freeze
+
+  module_function
+
+  def load_config
+    ThreeScale.config.currencies || DEFAULT_CURRENCIES
```

Comment:
This means that it only goes to the default if the currencies config is not set, otherwise, if they set the config and the dollar is not in their list, it won't be there... not sure if it is a good idea like this or not 🤷‍♀️